src/pages/task/taskTabs/logs/LogTypes.tsx

```diff
@@ -286,10 +286,11 @@ const useRenderBody: React.FC<{
 
 const ButtonContainer = styled.div`
   display: flex;
-  a:first-of-type {
+  > :not(:last-child) {
     margin-right: 8px;
   }
   margin-right: 16px;
+  padding-left: 1px;
```

Comment:
did you add this style so we can see left side shadow better?

Reply:
Exactly!